### PR TITLE
HRIS-279 [BE/FE] Modify Filing for Offset functionality

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Issue Link

- https://framgiaph.backlog.com/view/HRIS-279

## Definition of Done

- [x] Users can request an offset without changing the startTime/endTime
- [x] IsUsed column has been added on FiledOffsets modal

## Notes
- None

## Pre-condition
- In the root directory run ``` docker compose up db api client -d ```

## Expected Output
- In the filed offsets modal, it should reflect the isUsed status of the request
- It should reflect whether it is currently used or not by true or false

## Screenshots/Recordings

https://github.com/framgia/sph-hris/assets/109492180/3f5f41f2-0f22-4630-8ab7-3fda124350c0

### CODE REVIEW

File: client/src/components/molecules/FiledOffsetModal/columns.tsx [hunk 1]
```diff
@@ -33,6 +34,11 @@ export const columns = [
     footer: (info) => info.column.id,
     cell: (props) => <RequestStatusChip label={props.getValue()} />
   }),
+  columnHelper.accessor('isUsed', {
+    header: () => <CellHeader label="Is used" className="w-max !font-medium" />,
+    footer: (info) => info.column.id,
+    cell: (props) => <IsUsedChip label={props.getValue() ? 'true' : 'false'} />
```
Comment (jsvelte) [inline client/src/components/molecules/FiledOffsetModal/columns.tsx:40]: `props.row.original.isUsed ? 'true' : 'false'`

**Review by jsvelte — APPROVED** [review]
# Approved 🥇

**Review by arden-dave — APPROVED** [review]
Approve. 👍

**Review by JohnRoyCabezas — APPROVED** [review]

**Review by arden-dave — APPROVED** [review]
Approve. 👍

**Pull request merged**